argilla-frontend/components/features/annotation/container/questions/QuestionsForm.vue

```diff
@@ -179,7 +184,11 @@ export default {
         return "--focused-form";
     },
     formHasFocus() {
-      return this.autofocusPosition || this.autofocusPosition == 0;
```

Comment:
Consider simplifying this conditional logic to reduce nesting. An early return could enhance readability.